test_scripts/API/AutoCompleteList/005_ATF_SetGlobalProperties_P_AutoCompleteList_ValueSpaceBefore.lua

```diff
@@ -0,0 +1,79 @@
+----------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [SetGlobalProperties] Conditions for SDL SDL respond <success = true, resultCode = "SUCCESS"> to mobile app
```

Comment:
@Aleksandriys, please update summary with one more requirement, related to CRQ:
[SetGlobalProperties] SDL must transfer request to HMI in case of valid "autoCompleteList" param